src/package_manager/thm.rs

```diff
@@ -19,19 +21,21 @@ struct TreeHubPackage {
     pullUri: String,
 }
 
-pub fn install_package(path: &str) -> Result<InstallOutcome, InstallOutcome> {
+pub fn install_package<'t>(mtoken: Option<&Cow<'t, AccessToken>>, path: &str) -> Result<InstallOutcome, InstallOutcome> {
```

Comment:
This is arguable, but since this function requires an AccessToken, I would remove the Option and have the caller verify instead.